iosHyperskillApp/iosHyperskillApp/Sources/Views/SwiftUI/Styles/Buttons/PrimaryTextButtonStyle.swift

```diff
@@ -1,19 +1,18 @@
 import SwiftUI
 
-struct ProgressScreenCallToActionButtonStyle: ButtonStyle {
+struct PrimaryTextButtonStyle: ButtonStyle {
```

Comment:
You can delete this button style because it only has one difference from `ButtonStyle.automatic` with font:
1. automatic - body
2. here - subheadline

Reply:
@ivan-magda what about `.foregroundColor`? It definitely differs, `PrimaryTextButtonStyle` uses `Color(ColorPalette.primary)`. Or we should anyway remove it and use two modifiers for each such button (`.font`, `.foregroundColor`)

Reply:
> what about .foregroundColor? It definitely differs, PrimaryTextButtonStyle uses Color(ColorPalette.primary).

No, it's the same.

The app tint color is `ColorPalette.primary` and buttons automatically inherit it

https://github.com/hyperskill/mobile-app/blob/e14429e33b09eaa28480b62f479bbb36b51c1438/iosHyperskillApp/iosHyperskillApp/Sources/Systems/AppAppearance.swift#L5

Reply:
Got it :+1:. `themeApplication` function isn't used for previews, it confused me.